README.md

```diff
@@ -13,9 +13,26 @@ experience.
 ## Start a Reason React Native project with this template
 
 ```console
-npx @react-native-community/cli init MyApp --template @reason-react-native/template
+npx react-native init MyApp --template @reason-react-native/template
```

Comment:
I got issue with this command several times & got some report from other users. I don't like this change :/

Reply:
`npx react-native init MyApp` is the command recommended on the docs at [#using-npx-recommended](https://github.com/react-native-community/cli#using-npx-recommended) for what ever reason.

Reply:
yeah but it's just a proxy, read https://github.com/react-native-community/cli#about carefully.
in the past I used that recommendation, but in practise I had issue several times so I reverted & didn't get issues. Some people on discord got issues to if my memory is correct.